testbed/stack.ts

```diff
@@ -30,6 +30,15 @@ export class Testbed extends cdk.Stack {
                 },
             ],
         });
+
+        //Tag pub subnets for KIT CP
+        const selection = vpc.selectSubnets({
+            subnetType: ec2.SubnetType.PUBLIC
+        });
+        selection.subnets.forEach(subnet => {
+            Tags.of(subnet).add('kit/hostcluster', `${id}-controlplane`)
```

Comment:
Also will this be a single subnet in one AZ?
I thought about it a little bit more after our discussion and If thats the case, it will be not sufficient because CP nodes are created across different AZs (topology spread constraint)

Reply:
let's discuss offline what the issue is with KIT operator.
 Given megaVPC we can't have just one subnet per AZ as we need lots of IP's. Restricting only one subnet per AZ will limit us from running all megaxl tests that we want.